build.gradle

```diff
@@ -29,12 +33,15 @@ dependencies {
     testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '2.0.2'
     testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.2'
     testCompile group: 'org.scala-lang', name: 'scala-library', version: '2.12.10'
-    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '4.2.0'
+    testCompile('io.rest-assured:rest-assured:4.2.0') {
+        exclude group:'org.codehaus.groovy', module: 'groovy-xml'
+    }
```

Comment:
What does this change do?

Reply:
I'm glad you asked :smile: 

as you know, Gradle is built with Groovy, and once I've updated `intellij-idea gradle` plugin to the latest (4.2.1) it somehow added another version of groovy to the classpath. and as rest-assured turned out to also use gradle it led to some nasty errors and inability to compile.

the mentioned change explicitly excludes the version of `groovy-xml` that was ruining the party :balloon:

Reply:
I can't say if this is a correct way to handle this (perhaps it is) but as long as this is only about testing and tests works, I don't care too much.

Reply:
well, I'm not sure if it's a correct way either, but it works, does int make it correct :rofl: ?

Reply:
Well, I think it is possible that, after this change, some features of `rest-assured` do not work as expected but as this concerns only testing, it's enough that our tests pass. (If this was about production code, I would call for more thorough reasoning/scrutiny on the issue.)